discovery-infra/test_infra/consts.py

```diff
@@ -19,6 +19,15 @@
 DEFAULT_CLUSTER_KUBECONFIG_PATH = "build/kubeconfig"
 WAIT_FOR_BM_API = 900
 NAMESPACE_POOL_SIZE = 15
+DUMMY_SSH_KEY = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCj+lurtXW2WxojtNXSxEWWTmkm1VLzAj/" \
```

Comment:
Maybe this can move to test_cancel_and_reset.py I don't see other tests using this